airbyte-integrations/connectors/source-dockerhub/acceptance-test-config.yml

```diff
@@ -0,0 +1,26 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-dockerhub:dev
+tests:
+  spec:
+    - spec_path: "source_dockerhub/spec.yaml"
+  connection:
+    - config_path: "secrets/config.json"
```

Comment:
where these point to `secrets/config.json` should be replaced with `sample_files/config.json`.

Usually the config has to be in secrets due to it having secret values but that's not the case here so we can alter the pattern :)